packages/bar/index.d.ts

```diff
@@ -99,6 +99,7 @@ declare module '@nivo/bar' {
         padding: number
 
         valueScale: Scale
+        indexScale: Scale
```

Comment:
We should restrict to BandScale here, otherwise it might give the impression that other types are supported too.